testing/src/BazelTestPaths_TEST.cc

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <gtest/gtest.h>
+
+#include "ignition/common/Filesystem.hh"
+#include "ignition/common/testing/TestPaths.hh"
+#include "ignition/common/testing/BazelTestPaths.hh"
+
+using namespace ignition::common;
+
+char **kEnv;
+
+/////////////////////////////////////////////////
+TEST(TestPaths, DISABLED_DumpEnv)
```

Comment:
Not sure if disabling is a good idea here, this gets printed out when you run tests and looks like something to be fixed. I think if it were enabled it would just pass, right?

Reply:
Removed